modules/ROOT/pages/index.adoc

```diff
@@ -10,12 +10,12 @@ The MuleSoft approach to integration, https://blogs.mulesoft.com/dev/api-dev/wha
 
 image::api-led-architecture.png[Diagram of structured API layers for experience, process, and system APIs that can be easily added or removed]
 
-Anypoint Platform helps you build a structured application network that connects applications, data, and devices with reusable APIs.
-The unified Anypoint Platform makes it easy to discover, create, and manage APIs in a modular, organized layer. Instead of retrieving random and possibly unstable code snippets, you can “shop” for APIs created using the industry’s best practices.
+Anypoint Platform enables you to build a structured application network that connects applications, data, and devices with reusable APIs.
+Anypoint Platform makes it easy to discover, create, and manage APIs in a modular, organized layer. Instead of retrieving random, and possibly unstable code snippets, you can discover and use APIs that are created using the industry’s best practices.
 
 == Explore Anypoint Exchange
```

Comment:
Here and throughout, the use of an imperative verb in the title leads me to expect the steps of a Task topic, but none of these topics includes steps. I would replace the imperatives here with gerunds (-ing), making sure that the titles in the topics cross-referred to also use the proper verb case.